scripts/dev/tidy.php

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env php
-<?php
+<?php declare(strict_types=1);
```

Comment:
This reminded me that I have a POC to add strict types in our `run-tests.php` script as well. Gonna finish it and open for review 😊

Reply:
It's amazing addition. By just enabling this, the possible bugs get reduced a lot...

Reply:
The only thing I've noticed with this particular case - shebang and declaring strict types, that it  doesn't allow to include a script anymore in other files (for example for testing it)... I'm currently sort of going through the phase of how to effectively test this particular piece of file. And still trying to keep the one-file-cli script approach for simplicity... :/

Reply:
Not related to this change though, but probably the best way to test this script will be in a separate repository with a bunch of use cases and using some shell specific testing where output and exit codes are compared... I'm on this also so the script will be at least also pretty reliable... :)